src/Components/Resource/CommentSection.tsx

```diff
@@ -79,16 +80,16 @@ const CommentSection = (props: CommentSectionProps) => {
         onChange={(e) => setCommentBox(e.target.value)}
       />
       <div className="flex w-full justify-end">
-        <Button
+        <ButtonV2
           onClick={onSubmitComment}
-          className="border border-solid border-primary-600 hover:border-primary-700 text-primary-600 hover:bg-white capitalize my-2 text-sm"
+          className="bg-white border border-solid border-primary-600 hover:border-primary-700 text-primary-600 hover:bg-green-100 capitalize my-2 text-sm"
```

Comment:
@ayushjnv1 when using `ButtonV2` avoid using classNames unless necessary. Make use of `variant, border, ghost, circle` props to customize the design, that way we'll have consistent designs across the platform based on use case.

```suggestion
```